position_sets:
  set1: [0.0, 0.5, -0.5, 0.25]
  set2: [0.25, -0.5, 0.5, 0.0]
  set3: [-0.25, 0.0, 0.25, 0.5]
